.github/workflows/eslint.yml

```diff
@@ -0,0 +1,21 @@
+name: ESLint Widget v2
+
+on:
+  pull_request:
+    paths:
+      - 'packages/widget-v2/**'
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+      - name: Install dependencies
+        run: yarn install
+        working-directory: packages/widget-v2
+      - name: Run ESLint
+        run: yarn lint
+        working-directory: packages/widget-v2
```

Comment:
@codingki how do you feel about us adding this? it'll cause builds that have errors when running `yarn lint` to be red